services/crawler/services/IotexService.ts

```diff
@@ -0,0 +1,206 @@
+import Antenna from 'iotex-antenna'
+import { IActionInfo, IGetBlockMetasResponse, IGetChainMetaResponse, IGetReceiptByActionResponse, IGetServerMetaResponse } from 'iotex-antenna/lib/rpc-method/types'
+import { from } from '@iotexproject/iotex-address-ts'
+import { CandidateRegister, CandidateUpdate, StakeRestake, StakeWithdraw, StakeUnstake, StakeAddDeposit, ClaimFromRewardingFund, StakeCreate, StakeTransferOwnership } from 'iotex-antenna/lib/action/types'
+import { DepositToRewardingFund } from 'iotex-antenna/protogen/proto/types/action_pb'
+
+export interface IotexServiceOptions {
+  network: IotexNetworkType
+}
+
+export enum IotexNetworkType {
+  Mainnet = 'mainnet',
+  Testnet = 'testnet'
+}
+
+export interface IotexBlock {
+  type: string
+  id: string
+  height: number
+  datestring: string
+  producer: string
+  amount: string
+  transaction_root: string
+  num_of_actions: number
+}
+
+export interface IotexGovernanaceActions {
```

Comment:
`IotexActions` and we'll include the remaining action types following `IotexCreateStake`.